tams_ur5_setup_bringup/launch/apriltags2.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
```

Comment:
Please simplify this.
It starts the `apriltags2_ros_continuous_node` with our config in our setup.

- no need for arguments
- no need for the launch-prefix
- no need for explicit parameters that are false by default
- `<rosparam/>` can be put inside the `<node/>` xml-node so that you can drop the `ns` attribute.
- Is `output="screen"` necessary for everyday operation? If the user can see the same information somehow in RViz, this should not be in here either.

Reply:
Done!
At first I thought it might be useful to make these parameters configurable in case we use different camera settings (qhd/hd), tags etc in our ur5 setup. However for now it seems the easiest solution for such scenarios is to launch a dedicated apriltag2 node and drop the default launch file.